### ISSUE TITLE
Add child terms to midline glial cells [sf#20]

### ISSUE THREAD

**Issue by mmc46** [mmc46/drosophila-anatomy-developmental-ontology#20]
**Reported by mmc46 on 2013-10-23 18:42 UTC**
From Sian's email:
I'm looking at FBrf0213237, which appears to make some fine distinctions between anterior and posterior midline glial cells.  These are both presently exact synonyms of FBbt:00001315 "midline glial cell" (which has is_a children 'midline dorsal glial cell' and 'midline ventral glial cell').  The paper states (figure 1) that anterior and posterior midline glial cells differ in both gene expression and origin.  I wonder if this uniquely identifies anterior and posterior midline glial cells enough that they ought be split out as additional children of "midline glial cell".

**Comment by mmc46** [comment]
**Commented by mmc46 on 2013-12-03 12:24 UTC**
Hi Sian,
By stage 16, the midline glia is only formed by the anterior midline glia (AMG) identified at stage 13. The posterior midline glia from stage 13 degenerates and does not contribute to later stages. That is why 'midine glial cell' has the AMG synonym.
I'll leave the hierarchy as it is, but I've made some corrections to the synonyms (MGA, MGP, MGM). 
The term posterior midline glial cell isn't a synonym of midline glial cell, but if you were referring to MGP, that has now been corrected.

Thanks,
Marta
